server/controller/message/communityMessage.js

```diff
@@ -59,6 +60,7 @@ class CommunityMessageController {
      *
      */
   static async getMessage(req, res, next) {
+    const { limit, offset } = paginationUtil.paginate(req.query);
```

Comment:
```js
const { limit, offset } = req.query
const stuffYouNeed = paginationUtil.paginate(limit, offset);
```
It's much cleaner to pass exactly what the paginate function needs

Reply:
okay, got it.